### PR TITLE
Fix loadLana test

### PR DESCRIPTION
**Pull request by chrischrischris**
Moved the test into it's own file and call `loadLana` directly for testing.  When testing via scripts.test.js the lana test intermittently would fail.

**Test URLs:**

- https://loadlana--milo--adobecom.hlx.live/

### CODE REVIEW

**Review by hparra — APPROVED** [review]
Closing #986 was premature. There were other fixes in there.

**Comment by chrischrischris** [comment]
@hparra Thanks for pointing that out.  I've pulled those changes into this PR.

**Review by mokimo — APPROVED** [review]

File: test/blocks/bulk-publish/bulk-publish-utils.test.js [hunk 1]
```diff
@@ -126,7 +126,7 @@ describe('Bulk preview and publish', () => {
     localStorage.clear();
   });
 
-  it('Bulk preview URLs', async () => {
+  it.skip('Bulk preview URLs', async () => {
```
Comment (overmyheadandbody) [inline test/blocks/bulk-publish/bulk-publish-utils.test.js:129]: Is this still needed, is there any plan to enhance it or can it be removed entirely?

**Review by overmyheadandbody — APPROVED** [review]

File: test/blocks/bulk-publish/bulk-publish-utils.test.js (hunk 1 above)
Comment (jckautzmann) [reply test/blocks/bulk-publish/bulk-publish-utils.test.js:129]: @chrischrischris You mentioned that this test is intermittently failing. Can you please point me to links where I can see the failures? I don't see any failures when I run it locally.

Comment (chrischrischris) [reply test/blocks/bulk-publish/bulk-publish-utils.test.js]: I can't repro it now, so removing the `.skip`

Comment (mokimo) [reply test/blocks/bulk-publish/bulk-publish-utils.test.js]: Let's have an eye on it and ping @jckautzmann if we see it again so he can take a look

**Pull request merged**